.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     jest: true,
     node: true
   },
-  extends: ['plugin:vue/strongly-recommended', 'standard', 'prettier'],
+  extends: ['plugin:vue/strongly-recommended', 'plugin:vue/recommended', 'standard', 'prettier'],
```

Comment:
I think you can remove 'plugin:vue/strongly-recommended' : it is overrided by 'plugin:vue/recommended' and 'plugin:vue/recommended' contains the rules of 'plugin:vue/strongly-recommended'
https://github.com/vuejs/eslint-plugin-vue/blob/master/lib/configs/vue3-recommended.js